include/godot_cpp/classes/wrapped.hpp

```diff
@@ -226,15 +255,6 @@ public:
 		return m_inherits::get_class_static();                                                                                                                                         \
 	}                                                                                                                                                                                  \
                                                                                                                                                                                        \
-	static GDExtensionObjectPtr create(void *data) {                                                                                                                                   \
```

Comment:
These should also be removed from `GDEXTENSION_CLASS_ALIAS`, where they do nothing but also aren't used

Reply:
Awesome, I wasn't sure about those myself, but I can certainly remove those too, 👍